tools/jvm/src/main/scala/org/scalajs/core/tools/linker/backend/closure/LoggerErrorManager.scala

```diff
@@ -13,10 +13,12 @@ private[closure] class LoggerErrorManager(private val log: Logger)
     * package private.
     */
   override def report(level: CheckLevel, error: JSError): Unit = {
-    if (error.getType.key == "JSC_FRACTIONAL_BITWISE_OPERAND")
+    if (error.getType.key == "JSC_FRACTIONAL_BITWISE_OPERAND" ||
+        error.getType.key == "JSC_UNREACHABLE_CODE") {
```

Comment:
I guess there was no method of `options` for either of those, like we do [here](https://github.com/scala-js/scala-js/blob/608baf718e4366dfdcb0745f13facb2b438344f6/tools/jvm/src/main/scala/org/scalajs/core/tools/linker/backend/closure/ClosureLinkerBackend.scala#L137)?